manifests/README.md

```diff
@@ -0,0 +1,98 @@
+# Manifests
+
+This directory contains the manifests for deploying Boots to various environments.
+
+## KinD (Kubernetes in Docker)
+
+### Prerequisites
+
+- [KinD >= v0.12.0](https://kind.sigs.k8s.io/docs/user/quick-start#installation)
+- [Kubectl >= v1.23.4](https://www.downloadkubernetes.com/)
+
+### Steps
+
+1. Create KinD cluster
+
+   ```bash
+   # Create the KinD cluster
+   kind create cluster --config ./manifests/kind/config.yaml
+   ```
+
+2. Deploy Boots
+
+   ```bash
+   # Deploy Boots to KinD
+   kubectl kustomize manifests/kustomize/overlays/kind | kubectl apply -f -
+   ```
+
+3. Watch the logs
+
+   ```bash
+   kubectl -n tinkerbell logs -f $(kubectl get -n tinkerbell pods --template '{{range .items}}{{.metadata.name}}{{end}}' --selector=app=tinkerbell-boots)
```

Comment:
I think this (and other log watching) can be simplified to
```suggestion
   kubectl -n tinkerbell logs -f -l app=tinkerbell-boots
```

Reply:
oh excellent, will update. thanks